osconfig_tests/test_suites/guestpolicies/guest_policies.go

```diff
@@ -90,6 +98,43 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 	testSuite := junitxml.NewTestSuite(testSuiteName)
 	defer testSuite.Finish(testSuites)
 
+	storage, err := gcpclients.GetStorageClient()
```

Comment:
I would prefer the storage stuff to be in it's own function. It seemed to me like to do that it would need to return a cleanup function which would then be deffered but I'm on the fence as to whether that would be cleaner or not